HomeWork_1_nUnit/Divide.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HomeWork_1_nUnit
+{
+    [TestFixture]
+    public class Divide
+    {
+        Calculator calculation;
+
+        [OneTimeSetUp]
+        public void DivideSetup()
+        {
+            calculation = new Calculator();
+        }
+
+        [Test]
+        [Description("Divide values")]
+        [Category("Values")]
+        [Retry(3)]
+        public void DivideValues(
+            [Values(2, 16, 111)] int values1,
+            [Values(-5, -11)] int values2)
+        {
+            //Action
+            var actual = calculation.Divide(values1, values2);
+            var expected = values1 / values2;
+            Console.WriteLine($"First number: {values1};\nSecond number: {values2};\nActual divide result: {actual};\nExpected divide result: {expected};");
+            //Assert
+            Assert.AreEqual(expected, actual);
+        }
+
+        [Test]
+        [Description("Divide range")]
+        [Category("Range")]
+        [Retry(3)]
+        public void DivideRange(
+            [Values(6, 12)] int range1,
+            [Range(11, 13, 1)] int range2)
+        {
+            //Action
+            var actual = calculation.Divide(range1, range2);
+            var expected = range1 / range2;
+            Console.WriteLine($"First number: {range1};\nSecond number: {range2};\nActual divide result: {actual};\nExpected divide result: {expected};");
+            //Assert
+            Assert.AreEqual(expected, actual);
+        }
+
+        [Test]
```

Comment:
Also you can simplify like

[Test, Category("Category"), Description("Description")]